src/modules/views/analytics/AnalyticsView.tsx

```diff
@@ -0,0 +1,75 @@
+import { useTranslation } from 'react-i18next';
+
+import { Alert, Box } from '@mui/material';
+
+import { AppAction } from '@graasp/sdk';
+import { Loader } from '@graasp/ui';
+
+import { differenceInSeconds } from 'date-fns';
+import groupBy from 'lodash.groupby';
+import orderBy from 'lodash.orderby';
+
+import { APP_ACTIONS_TYPES } from '@/config/appActionsTypes';
+import { hooks } from '@/config/queryClient';
+import { CodeVersionType } from '@/interfaces/codeVersions';
+
+import { ANALYTICS_VIEW_CY } from '../../../config/selectors';
+import GeneralStatistics from './GeneralStatistics';
+import UsersRunningCodeVersions from './UsersRunningCodeVersions';
+
+const AnalyticsView = (): JSX.Element => {
+  const { data, isLoading } = hooks.useAppActions();
+
+  const { t } = useTranslation();
+
+  if (data) {
+    const actionsOrdersByCreatedDate = orderBy(data, 'createdAt');
```

Comment:
If you sort ISO date string lexicographically (alphabetically) does it result in a chronological sort ?

Reply:
Yes, It seems that under the hood it convert date to iso format,
SO it worked magically